bazel/README.md

```diff
@@ -80,6 +80,9 @@ for how to update or override dependencies.
     echo "build --config=clang" >> user.bazelrc
     ```
 
+    Note: `libstdc++-7-dev` (or higher) must be installed. This is typically available via a
```

Comment:
This isn't true if you're using libc++, the alternative is use `--config=libc++`.

Reply:
So should it say install libstdc++-7-dev or use `--config=libc++`?

Reply:
Yes